app/src/main/java/com/epfl/neighborfood/neighborfoodandroid/ui/fragments/VendorDashboardFragment.java

```diff
@@ -0,0 +1,90 @@
+package com.epfl.neighborfood.neighborfoodandroid.ui.fragments;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.epfl.neighborfood.neighborfoodandroid.R;
+import com.epfl.neighborfood.neighborfoodandroid.adapters.MealListAdapter;
+import com.epfl.neighborfood.neighborfoodandroid.adapters.OrderListAdapter;
+import com.epfl.neighborfood.neighborfoodandroid.databinding.FragmentVendorDashboardBinding;
+import com.epfl.neighborfood.neighborfoodandroid.models.Meal;
+import com.epfl.neighborfood.neighborfoodandroid.ui.activities.PlaceMealActivity;
+
+import java.sql.Array;
+import java.util.ArrayList;
+
+public class VendorDashboardFragment extends Fragment {
+
+    private FragmentVendorDashboardBinding binding;
+    private ArrayList<String> nm;
+    private ArrayList<Integer> id;
+
+    @Override
+    public View onCreateView (LayoutInflater inflater,
+                              ViewGroup container, Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        binding = FragmentVendorDashboardBinding.inflate(getLayoutInflater());
+        return binding.getRoot();
+
+    }
+    @Override
+    public void onViewCreated(View view, Bundle savedInstanceState){
+
+        id = new ArrayList<Integer>();
+        id.add(R.drawable.poulet);
+        id.add(R.drawable.couscous);
+        id.add(R.drawable.paella);
+
+        nm = new ArrayList<String>();
+        nm.add("Poulet au miel");
+        nm.add("Couscous aux légumes");
+        nm.add("Paella aux crevettes");

```

Comment:
you can make this as as a HashMap with ids as keys and names as values so it can be better